packages/lib/BaseApplication.ts

```diff
@@ -927,6 +929,9 @@ export default class BaseApplication {
 
 		await MigrationService.instance().run();
 
+		const rotationalLogs = new RotatingLogs(profileDir);
+		await rotationalLogs.rotateLogsFiles();
```

Comment:
Could you make rotationalLogs a property of the class? Then call it like once 60 seconds after the app starts, then again once every 24 hours. See how it's done for the auto-update check:

```typescript
			// Initial check on startup
			shim.setTimeout(() => { runAutoUpdateCheck(); }, 5000);
			// Then every x hours
			shim.setInterval(() => { runAutoUpdateCheck(); }, 12 * 60 * 60 * 1000);
```

We do it that way so that the app doesn't take too long to start.

You will want to try/catch errors at this point, and log them